src/basic/user-util.c

```diff
@@ -465,6 +465,54 @@ int in_group(const char *name) {
         return in_gid(gid);
 }
 
+static
+bool gid_list_has(const gid_t *list1, int size1, gid_t val) {
```

Comment:
please follow our usual coding style, i.e. no line-break after the static please

Reply:
why the "1" suffix?

also, we generally try to stick to size_t for counting memory objects, please no int here.